include/networkit/algebraic/Vector.hpp

```diff
@@ -309,7 +309,7 @@ Matrix Vector::outerProduct(const Vector &v1, const Vector &v2) {
         for (index j = 0; j < v2.getDimension(); ++j) {
             double result = v1[i] * v2[j];
             if (fabs(result) >= FLOAT_EPSILON) {
-                triplets.push_back({i, j, result});
+                triplets.emplace_back(i, j, result);
```

Comment:
Unrelated change?

Reply:
If I do not change this then the clang-tidy build fails due to `modernize-use-emplace`.

Reply:
Why doesn't clang-tidy fail right now?